backend/organization/views/project_views.py

```diff
@@ -219,6 +221,13 @@ def patch(self, request, url_slug, format=None):
             project = Project.objects.get(url_slug=url_slug)
         except Project.DoesNotExist:
             return Response({'message': 'Project not found: {}'.format(url_slug)}, status=status.HTTP_404_NOT_FOUND)
+        
+        # Author: Dip
+        # Code formatting here. So fields are just pass through so combing them and using setattr method insted.
+        pass_through_params = ['collaborators_welcome', 'description', 'helpful_connections','short_description', 'website']
+        for param in pass_through_params:
+            if param in request.data:
+                setattr(project, param, request.data[param])
```

Comment:
Didn't you report that there was an issue with this logic? Is it still happening?

Reply:
I think I fixed that. As far as I remember the problem was that it was trying access `request.data.param` rather than `request.data[param]`